docs/rnw-dependencies.md

```diff
@@ -27,33 +27,37 @@ Start an **elevated** PowerShell window and run:
 
 Alternatively, you can setup your environment manually:
 - Ensure Developer Mode is turned ON in Windows Settings App.
-- Install [Visual Studio 2019 (version 16.5 or greater)](https://www.visualstudio.com/downloads) **with the following options checked**:
-  - Workloads
+- It is _highly_ recommended to update the Windows system.
+- Install a recent version of [Visual Studio 2019](https://www.visualstudio.com/downloads) **with the following options checked**:
+  - **Workloads**
     - Universal Windows Platform development
       - Include `C++ (v142) Universal Windows Platform tools` (under 'Optional')
+      - Older Windows 10 SDK version may be needed at this point.
```

Comment:
This statement is a more fuzzy version of what I proposed in PR #443.

I think this statement is needed at least until the proposal in <https://github.com/microsoft/react-native-windows/pull/7779> is merged and released.

Even after <https://github.com/microsoft/react-native-windows/pull/7779> is released, I think this statement would be needed in case Visual Studio suddenly updates the default Windows 10 SDK version again.